tests/dto/risk-score-api.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { expect, test } from '@playwright/test'
+import { StatusCodes } from 'http-status-codes'
+import { RiskDto } from './risk-dto'
+
+test('Calculate risk score with valid data for Low Risk level - positive decision - receive code 200', async ({
+  request,
+}) => {
+  const response = await request.post(
+    'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+    {
+      data: RiskDto.calculateRiskScore(),
+    },
+  )
+
+  console.log('response status:', response.status())
+  console.log('response body:', await response.json())
+  expect.soft(response.status()).toBe(StatusCodes.OK)
+})
+
+test('Calculate risk score with valid data for Medium risk level- positive decision- receive code 200', async ({
+  request,
+}) => {
+  const riskDto = RiskDto.calculateRiskScore()
+  riskDto.loanPeriod = 9
+  const response = await request.post(
+    'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+    {
+      data: riskDto,
+    },
+  )
+  console.log('response status:', response.status())
+  console.log('response body:', await response.json())
+  expect.soft(response.status()).toBe(StatusCodes.OK)
```

Comment:
Verify a response code, and **all fields** in the response using soft asserts

Reply:
added to all tests all fields